crates/cli/tests/cli_run.rs

```diff
@@ -864,6 +864,7 @@ mod cli_run {
     }
 
     #[test]
+    #[serial(parser_package)]
```

Comment:
Multiple apps using the same package simultaneously can lead to [problems](https://github.com/roc-lang/roc/actions/runs/4210389674/jobs/7308770878).